common/concurrency/WorkerPool.h

```diff
@@ -11,6 +11,11 @@ class WorkerPool {
         using namespace std::chrono_literals;
         return 250ms;
     }
+    // [IDE] We want blocking operations to be more responsive to preemption requests.
+    inline static constexpr std::chrono::milliseconds PREEMPTION_BLOCK_INTERVAL() {
+        using namespace std::chrono_literals;
+        return 20ms;
```

Comment:
are there any downsides of changing the other value from 250ms to 20ms?

Reply:
Good question. I can time some runs on pay-server to see if it impacts throughput.

Reply:
Nope! Perf seems unaffected running typecheck on pay-server.